browser/src/AcOfOnePage.tsx

```diff
@@ -0,0 +1,193 @@
+import React from 'react'
+import styled from 'styled-components'
+import { ExternalLink, ListItem, OrderedList, PageHeading } from '@gnomad/ui'
+import { PaperCitation } from './PublicationsPage'
+
+// @ts-expect-error
```

Comment:
There are some tricks I've heard of lately to allow you to import arbitrary file types into TS, can you see if something like [this](https://graffino.com/til/how-to-import-images-in-typescript) works and if so let's do that rather than add in a load of new `ts-expect-error`?